server/src/router.ts

```diff
@@ -315,3 +342,34 @@ const spawnRates: { [timeSeconds: number]: number } = {
   80: 0.6,
   100: 0.7,
 };
+
+const removeImageAfterSecs = (secondsImageShown: Number) => {
+  backgroundState.isShowingImage = true;
+
+  if (backgroundState.imageInterval) {
+    clearInterval(backgroundState.imageInterval);
+    backgroundState.loopCounter = 0;
+  }
+
+  backgroundState.imageInterval = setInterval(() => {
```

Comment:
use a setTimeout instead of setInterval since it seems like you only need to wait one minute instead of repeating an action every second. tower defense uses an interval to mimic a game loop, where you could do

```
setTimeout(() => { ...removeImage()...}, 60000)